key_action_map.cpp

```diff
@@ -310,4 +386,22 @@ void test_key_action_map()//!OCLINT tests can be many
     assert(kam.to_key(action_type::shoot_stun_rocket) == sf::Keyboard::E);
   }
 #endif // FIX_ISSUE_355
+
+//#define ISSUE_522
+#ifdef ISSUE_522
+  {
+    const key_action_map kam = get_player_1_kam();
+    std::stringstream s;
+    s << kam;
+  }
+  {
+    const key_action_map kam = get_player_1_kam();
+    const std::string filename = "test.txt";
+    save_to_file(kam, filename);
+    const key_action_map map_again = load_kam(filename);
+    assert(kam == map_again);
```

Comment:
This test is failing, need to figure out what is wrong here

Reply:
Ah, my bad, thanks for rejecting this PR: I would have messed up develop :-)

Reply:
You disabled the tests, so develop won't break if we merge this PR. I was playing a bit with the tests to see whether your code works, if we are going to leave the issues to the new devs, then I will disable them again

Reply:
Ah, I guess you found out why I disabled that test :-)